pyupgrade.py

```diff
@@ -745,6 +746,9 @@ def _fix_tokens(contents_text, min_version):
             del tokens[i]
             if tokens[i].name == 'NL':  # pragma: no branch (old PY2)
                 del tokens[i]
+                # pragma: no branch (old PY2)
+                if blank_re.match(tokens[i].src):
+                    del tokens[i]
```

Comment:
I think technically there should be another nested check for `if tokens[i].name == 'NL':` here, but I guess the leading empty line trimming takes care of it? Seems to work anyway.